src/main/java/com/bringframework/exceptions/BeanInjectionException.java

```diff
@@ -0,0 +1,8 @@
+package com.bringframework.exceptions;
+
```

Comment:
Please add documentation here, cause for all exceptions classes we have java doc

Reply:
fixed